tsconfig.json

```diff
@@ -1,22 +1,26 @@
 {
   "compilerOptions": {
-    "target": "esnext",
+    "target": "es2018",
     "module": "esnext",
     "strict": true,
     "jsx": "preserve",
     "importHelpers": true,
     "moduleResolution": "node",
+    "skipLibCheck": true,
```

Comment:
Maybe we can double check these changes?

Maybe we can delete this root tsconfig for good.